garnahata_site/catalog/static/js/garnahata.js

```diff
@@ -110,4 +116,45 @@
         detail.map.addLayer(markers);
         detail.map.fitBounds(markers)
     });    
+    
+    $(document).ready(function () {
+        $('.objectAdressTable').each(function () {
+            var Column_number_to_Merge = 1;
+ 
+            // Previous_TD holds the first instance of same td. Initially first TD=null.
+            var Previous_TD = null;
```

Comment:
I'll accept that for now, but cannot you implement that right in the template, using logic of Jinja2?